examples/DynaSwing/IEC/Wind/Neplan/WPP4ACurrentSource2015Q/WPP4ACurrentSource2015.par

```diff
@@ -211,6 +211,6 @@
     <par type="DOUBLE" name="minStep" value="1e-7"/>
     <par type="DOUBLE" name="maxStep" value="10"/>
     <par type="DOUBLE" name="absAccuracy" value="1e-7"/>
-    <par type="DOUBLE" name="relAccuracy" value="1e-4"/>
+    <par type="DOUBLE" name="relAccuracy" value="9e-5"/>
```

Comment:
This value for relative accuracy is not recommended (and quite odd :)) Let me try to find something else to stabilize this test